src/ui/display_nbgl.c

```diff
@@ -540,7 +540,7 @@ void ui_display_erc4361_content_flow(void) {
     nbgl_useCaseReview(TYPE_MESSAGE,
                        &pairList,
                        &C_Acre_64px,
-                       "Review ERC4361 message",
+                       "Review Sign In message",
```

Comment:
Related to the other comment. Can we update the text?
```suggestion
                       "Review Sign In",
```